pylabrobot/resources/celltreat/tubes.py

```diff
@@ -0,0 +1,19 @@
+from pylabrobot.resources.tube import Tube
+
+
+def celltreat_15000ul_centrifuge_tube_Vb(name: str) -> Tube:
+  """CELLTREAT® Centrifuge Tubes-RackMaster 15mL Centrifuge Tube, Best Value - Paperboard Rack, Sterile
+  Part no.: 229414
+
+  - bottom_type=TubeBottomType.V
+  """
```

Comment:
this as well. could you please link a drawing or comment these values are measured?